File: 2-semestre/uri-online-judge/javascript/1049.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

let palavra1 = lines.shift();
let palavra2 = lines.shift();
let palavra3 = lines.shift();

if (palavra1.localeCompare("vertebrado") === 0) {
    if (palavra2.localeCompare("ave") === 0) {
        if (palavra3.localeCompare("carnivoro") === 0) {
            console.log("aguia");
        } else if (palavra3.localeCompare("onivoro") === 0) {
            console.log("pomba");
        }
    } else if (palavra2.localeCompare("mamifero") === 0) {
        if (palavra3.localeCompare("onivoro") === 0) {
            console.log("homem");
        } else if (palavra3.localeCompare("herbivoro") === 0) {
            console.log("vaca");
        }
    }
} else if (palavra1.localeCompare("invertebrado") === 0) {
    if (palavra2.localeCompare("inseto") === 0) {
        if (palavra3.localeCompare("hematofago") === 0) {
            console.log("pulga");
        } else if (palavra3.localeCompare("herbivoro") === 0) {
            console.log("lagarta");
        }
    } else if (palavra2.localeCompare("anelideo") === 0) {
        if (palavra3.localeCompare("hematofago") === 0) {
            console.log("sanguessuga");
        } else if (palavra3.localeCompare("onivoro") === 0) {
            console.log("minhoca");
        }
    }
}

File: 2-semestre/uri-online-judge/c/1049.c
#include <stdio.h>
#include <string.h> 

int main() {
    char palavra1[15];
    char palavra2[15];
    char palavra3[15];
        
    scanf("%s", palavra1);
    scanf("%s", palavra2);
    scanf("%s", palavra3);
        
    if (0 == strcmp(palavra1, "vertebrado")) {
        if (0 == strcmp(palavra2, "ave")) {
            if (0 == strcmp(palavra3, "carnivoro")) {
                printf("aguia\n");
            } else if (0 == strcmp(palavra3, "onivoro")) {
                printf("pomba\n");
            }
        } else if (0 == strcmp(palavra2, "mamifero")) {
            if (0 == strcmp(palavra3, "onivoro")) {
                printf("homem\n");
            } else if (0 == strcmp(palavra3, "herbivoro")) {
                printf("vaca\n");
            }
        }
    } else if (0 == strcmp(palavra1, "invertebrado")) {
        if (0 == strcmp(palavra2, "inseto")) {
            if (0 == strcmp(palavra3, "hematofago")) {
                printf("pulga\n");
            } else if (0 == strcmp(palavra3, "herbivoro")) {
                printf("lagarta\n");
            }
        } else if (0 == strcmp(palavra2, "anelideo")) {
            if (0 == strcmp(palavra3, "hematofago")) {
                printf("sanguessuga\n");
            } else if (0 == strcmp(palavra3, "onivoro")) {
                printf("minhoca\n");
            }
        }
    }
 
    return 0;
}
